src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -11,17 +11,34 @@ static class QueuePermissions
         public static void CheckQueue(string address)
         {
             var msmqAddress = MsmqAddress.Parse(address);
+            var queuePath = msmqAddress.PathWithoutPrefix;
 
-            MessageQueue messageQueue;
-            if (!MsmqUtilities.TryOpenQueue(msmqAddress, out messageQueue))
+            Logger.Debug($"Checking if queue exists: {queuePath}.");
+
+            if (msmqAddress.IsRemote)
             {
-                Logger.Warn($"Unable to open the queue at address '{address}'. Make sure the queue exists, and the address is correct. Processing will still continue.");
-                return;
+                Logger.Debug("Queue is on remote machine.");
+                Logger.Debug("If this does not succeed (like if the remote machine is disconnected), processing will continue.");
```

Comment:
although this is just copy&pasted from above, I think we should improve the logging on this condition. Why do we need to log entries for something like that and the second entry doesn't seem to be helpful in any way to a user?